components/business-adaptors/sap/src/main/java/org/wso2/carbon/transports/sap/SAPTransportSender.java

```diff
@@ -168,29 +180,24 @@ public void sendMessage(MessageContext messageContext, String targetEPR,
                     OMElement payLoad,body;
                     body = messageContext.getEnvelope().getBody();
                     payLoad = body.getFirstChildWithName(new QName(RFCConstants.BAPIRFC));
-                    if (log.isDebugEnabled()){
-                        log.debug("Received RFC/Meta DATA: " + payLoad);
-                    }
-                    String rfcFunctionName = RFCMetaDataParser.getBAPIRFCFucntionName(payLoad);
+                    log.info("Received RFC/Meta DATA: " + payLoad);
+                    String rfcFunctionName = RFCMetaDataParser.getBAPIRFCFunctionName(payLoad);
+                    log.info("RFC name: " + rfcFunctionName);
                     if (isTransaction(messageContext) || isLogon) {
-                        log.info("Begin Transaction");
+                        if (log.isDebugEnabled()) {
+                            log.debug("Beginning Transaction");
+                        }
                         JCoContext.begin(destination);
+                        log.info("Transaction begun successfully");
                     }
                     if (isLogon) {
                         logon(messageContext, destination, escapeErrorHandling);
                     }
-                    if (log.isDebugEnabled()){
-                        log.debug("Looking up the BAPI/RFC function: " + rfcFunctionName + ". In the " +
-                                "meta data repository");
-                    }
+                    log.info("Looking up the BAPI/RFC function: " + rfcFunctionName + ". In the " +
```

Comment:
What's the reason for make this log into an info log?

Reply:
done